index.html

```diff
@@ -274,5 +277,69 @@ <h2 class="about-me-header">About me</h2>
             </ul>
         </div>
     </section>
+    <!-- contact section-->
+    
+    <section class="contact-section">
+        <h2>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/mgebdyjo"
+          
+          method="POST"
+          id="contact-form"
+          class="contact-form"
+        >
+          <fieldset>
+            <input
+              class="inputs"
+              type="text"
+              name="full-name"
+              id="fullname"
+              maxlength="30"
+              placeholder="full name"
+              required
+            />
+            <input
+              class="inputs"
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Email address"
+              required
+            />
+            <textarea
+              name="comments"
+              id="comments"
+              maxlength="500"
+              placeholder="Enter text here"
+            ></textarea>
```

Comment:
- [ ] Kindly please add the `required` attribute in your *textarea* because without it any user can send a mistake sending an email to you which actually will result in a waste of resources. Please look at the below example.

> Suppose you have a website which sells some products and people have some queries before buying any product in order to get the leads and get an instant email from the customer you will need better email services which cost $10 per 1000 emails. Now a genuine user definitely asks the relevant things and maybe your customer gets converted and ready to buy your product then in that case your expense on email services will consider valuable but in the worst-case scenario, a user who just playing with your website and miss-use your resources will decrease your sales and also put unnecessary bill on you.  


*I hope the above example helps you why we need to require the `required` attribute in the form.*